pkg/util/platformworkloadidentity/federatedcredentials_test.go

```diff
@@ -29,7 +29,7 @@ func TestGetPlatformWorkloadIdentityFederatedCredName(t *testing.T) {
 	})
 
 	t.Run("has expected key as prefix", func(t *testing.T) {
-		wantPrefix := fmt.Sprintf("%s-%s-%s", subscriptionId, resourceGroup, clusterName)
+		wantPrefix := fmt.Sprintf("%s_%s", clusterName, strings.Join((strings.Split(strings.ReplaceAll(saName, ":", "-"), "-"))[2:], "-"))
```

Comment:
Consider being explicit here rather than duplicating the strings logic.
```suggestion
		wantPrefix := fmt.Sprintf("%s_%s", clusterName, "openshift-cloud-controller-manager")
```